.gitignore

```diff
@@ -1,2 +1,3 @@
 /out
 /cache
+.vscode/
```

Comment:
No need to add these IDE-specific files to a .gitignore. These should be defined to always be ignoring in your global gitignore file in your home directory.